### ISSUE TITLE
Start location is the left most (minimum) value but sometimes left most is not the minimum

### ISSUE THREAD

**Issue by bluesurfer** [biopython/biopython#1969]
Found myself in the special case of a wrapping origin feature.
```
from Bio.Seq import Seq
from Bio.SeqFeature import SeqFeature, FeatureLocation

feature = SeqFeature(FeatureLocation(90, 100) + FeatureLocation(0, 20))
```
where 100 is the length of the sequence. 

In this case `feature.location.start` should be the left most value of the feature i.e. 90. However Biopython is picking the minimum value i.e. 0 and that does not correspond to the left most value. 

Is there a reason why Biopython does not simply pick the start of a location as the first element of its first part? Something like:
```
feature.location.parts[0].start 
```
The same goes for the end.

**Comment by peterjc** [comment]
```python
>>> feature.location.start
ExactPosition(0)
>>> feature.location.end
ExactPosition(100)
>>> list(feature.location)
[90, 91, 92, 93, 94, 95, 96, 97, 98, 99, 0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19]
```

This is by design as as documented "*Start location - left most (minimum) value, regardless of strand.*",

https://github.com/biopython/biopython/blob/biopython-173/Bio/SeqFeature.py#L1390

Have you read the ``FeatureLocation`` class docstring?

"*i.e. Rather than thinking of the 'start' and 'end' biologically in a
    strand aware manor, think of them as the 'left most' or 'minimum'
    boundary, and the 'right most' or 'maximum' boundary of the region
    being described. This is particularly important with compound
    locations describing non-continuous regions.*"

https://github.com/biopython/biopython/blob/biopython-173/Bio/SeqFeature.py#L666

**Comment by peterjc** [comment]
I don't understand what you mean by 'left most', and why it might be a useful thing to return. We are returning the left-most or minimum value (0) on this example, which is useful in the context of drawing genome diagrams.

You do not seem to want the biological start/end, see https://github.com/biopython/biopython/issues/897

**Comment by bluesurfer** [comment]
Just to clarify, I am not issuing a bug but possibly an enhancement. 

Precisely, the one reported in #897. Since one may be working in different contexts it may be useful to have the opportunity to choose between biological start/end and left/right of the location.

To me this issue may be closed in favor of #897

**Comment by peterjc** [comment]
Thank you for clarifying - another vote in favour of #897 👍 

As noted there, we were stuck with the historic behaviour labels of ``.start`` and ``.end`` (which in hindsight could have been ``.left`` and ``.right``), which constrained how the relatively recent ``CompoundLocation`` could behave.
